WorkAndFun/src/main/java/pl/infoshare/Menu.java

```diff
@@ -0,0 +1,53 @@
+package pl.infoshare;
+
+import java.util.Scanner;
+
+public class Menu {
+    public void menu() {
+        System.out.println("----------------------------------------------------");
+        System.out.println("&&&&&&&&&&&--->> Work & Fun Service <<---&&&&&&&&&&&");
+        System.out.println("----------------------------------------------------");
+        System.out.println("Wybierz jedna z opcji wprowadzajac odpowiednia cyfre");
+        System.out.println("1 - Wyszukaj ogloszenie z oferowana usluga");
+        System.out.println("2 - Wyszukaj ogloszenie z poszukiwana usluga");
+        System.out.println("3 - Dodaj ogloszenie z oferowana usluga");
+        System.out.println("4 - Dodaj ogloszenie z poszukiwana usluga");
+        System.out.println("5 - Edytuj ogloszenie");
+        System.out.println("6 - Usun ogloszenie");
+        System.out.println("7 - Zakoncz program");
+        System.out.println("----------------------------------------------------");
+        getMenuChoice();
+    }
+
+    public void getMenuChoice() {
+        Scanner scanner = new Scanner(System.in);
+        String string = scanner.nextLine();
```

Comment:
Ja bym nie nazywał zmiennej "string". Może ta zmienna nigdzie węcej nie będzie używana, ale to po prostu nie jest zgodne z zasadami clean code, a proponuję żebyśmy pisali zawsze jak najlepiej się da i przyzwyczajać się do tego.